pkg/model/serializer.go

```diff
@@ -46,6 +46,7 @@ type buildInfoRaw struct {
 	VolumesToInclude []StackVolume  `yaml:"-"`
 	ExportCache      string         `yaml:"export_cache,omitempty"`
 	DependsOn        BuildDependsOn `yaml:"depends_on,omitempty"`
+	Secrets          BuildSecrets   `yaml:"secrets,omitempty"`
```

Comment:
Can we add a comment to this struct like

"it's an auxiliary struct that should match `BuildInfo` and it's used to handle recursion in custom un/marshalling"

So we understand why this struct has to change when `BuildInfo` changes

Reply:
i think this is self-explanatory taking into account this is a private struct used under the Unmarshall functions